src/routers/categories.js

```diff
@@ -1,19 +1,26 @@
 const categoriesRouter = require('express').Router();
 const { CategoriesController } = require('../controllers');
 const { categoryPayloadValidator } = require('../middlewares/validators');
+const { verifyRole } = require('../middlewares/auth');
 
 const categoriesController = new CategoriesController();
 categoriesRouter.post(
   '/',
+  verifyRole('Admin', 'Super admin'),
```

Comment:
Put this middleware before your https://github.com/Sf-62-NodeJS/clothes-market-server/blob/main/src/middlewares/auth/checkAuth.js

Reply:
Fixed. BTW, I couldn't create tests for this.